src/posix/platform/system.cpp

```diff
@@ -66,10 +66,51 @@ static void processStateChange(otChangedFlags aFlags, void *aContext)
 }
 #endif
 
+static const char *get802154RadioUrl(otPlatformConfig *aPlatformConfig)
+{
+    const char *radioUrl = nullptr;
+
+    for (uint8_t i = 0; i < aPlatformConfig->mRadioUrlNum; i++)
+    {
+        ot::Posix::RadioUrl url(aPlatformConfig->mRadioUrls[i]);
+        if (strcmp(url.GetProtocol(), "trel") == 0)
```

Comment:
nit: add empty line after declaration
```suggestion
        ot::Posix::RadioUrl url(aPlatformConfig->mRadioUrls[i]);

        if (strcmp(url.GetProtocol(), "trel") == 0)
```